src/lib/decorations/brace.typ

```diff
@@ -57,145 +61,85 @@
 ///
 /// - start (coordinate): Start point
 /// - end (coordinate): End point
-/// - flip (bool): Flip the brace around
 /// - name (string, none): Element name used for querying anchors
 /// - ..style (style): Style key-value pairs
-#let brace(
-  start,
-  end,
-  flip: false,
-  debug: false,
-  name: none,
-  ..style,
-) = {
-  // Validate coordinates
-  let t = (start, end).map(coordinate.resolve-system)
+#let brace(start, end, ..style, name: none) = group(name: name, ctx => {
+  assert.eq(style.pos().len(), 0,
+    message: "Brace takes no additional positional arugments.")
+
+  // Resolve all coordinates
+  let (ctx, start, end) = coordinate.resolve(ctx, start, end)
```

Comment:
assert and coordinate checks should be outside of the returned function

Reply:
Fixed.